src/cider/nrepl/middleware/util/java/parser.clj

```diff
@@ -243,8 +243,10 @@
 
 (defn source-path
   "Return the relative `.java` source path for the top-level class."
-  [class]
-  (-> (str/replace (str class) #"\$.*" "")
+  [cls]
```

Comment:
Why did you rename this?

Reply:
I don't like let-bound names shadowing existing names, `clojure.core/class` in this case. This limits introspection.

Reply:
Got it. I think a more common approach is to use the name `klass`, though. :-) Something like this should be in a separate commit I think.